charts/cert-manager-monitoring/templates/prometheus/prometheusrule.yaml

```
@@ -44,6 +44,22 @@ spec:
       for: 1h
       labels:
         severity: warning
+    - alert: CertManagerCertExpirySoon
```

Should we give it a distinct name? 
```suggestion
    - alert: CertManagerCertExpiryVerySoon
```

---

Having different names will result in two alerts being open at the same time. AFAIK Prometheus best practices is to simply raise the severity of the alert but keep the same name, e.g. check [KubePersistentVolumeFillingUp](https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/af5e89820645ab7f99c8be0c247ab2f9f845869d/alerts/storage_alerts.libsonnet#L19-L68) for reference.